test/protocol/clients/ClientExternalAddreses.js

```diff
@@ -1,16 +1,17 @@
 const hre = require("hardhat");
```

Comment:
I think this file should be renamed `ClientExternalAddressesTest.js` to match the pattern of al the other tests.

Reply:
done